frontend/src/components/core/Maybe/Maybe.test.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * @license
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react"
+import { mount, ReactWrapper } from "enzyme"
+import Maybe from "./Maybe"
+
+interface OuterProps {
+  name: string
+}
+
+interface InnerProps {
+  name: string
+}
+const Inner = (props: InnerProps): any => <div>{props.name}</div>
+
+describe("The Maybe component", () => {
+  describe("when enable is true", () => {
+    let component: ReactWrapper
+
+    beforeEach(() => {
+      const Outer = (props: OuterProps): any => (
+        <Maybe enable={true}>
+          {" "}
```

Comment:
???

Reply:
oh... i need to remove this... thanks for catching it :-)